utils.ts

```diff
@@ -522,12 +522,18 @@ export async function applyPackageOverrides(
 	await overridePackageManagerVersion(pkg, pm)
 
 	if (pm === 'pnpm') {
+		const overridesWithoutSpecialSyntax = Object.fromEntries(
+			Object.entries(overrides)
+				//eslint-disable-next-line @typescript-eslint/no-unused-vars
+				.filter(([key, value]) => (value as string).includes('>')),
```

Comment:
This should work if the rule is configured to allow _ prefix for unused vars
```suggestion
				.filter(([_key, value]) => (value as string).includes('>')),
```